.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,147 @@
+name: "Calendar Bug"
+description: "Have you encountered a bug?"
+labels: ["bug", "0. Needs triage"]
```

Comment:
compare with https://github.com/nextcloud/calendar/labels. the triage label has a different name.

Reply:
ok, anything else to change?